src/ImportDefinitionsBundle/Resources/config/pimcore/config.yml

```diff
@@ -61,4 +62,4 @@ import_definitions:
             fetcher_abstract: '/bundles/importdefinitions/pimcore/js/fetchers/abstract.js'
             fetcher_objects: '/bundles/importdefinitions/pimcore/js/fetchers/objects.js'
         css:
-            import_definition: '/bundles/importdefinitions/pimcore/css/importdefinition.css'
\ No newline at end of file
+            import_definition: '/bundles/importdefinitions/pimcore/css/importdefinition.css'
```

Comment:
This project needs `.editorconfig` and `.gitattributes` :)

Reply:
PR? :)